### PR TITLE
Add pod labels

### PR DESCRIPTION
**Pull request by cmanzur**
Add custom extra pod labels.
Example: Useful for k8s logging shipping.

### CODE REVIEW

File: charts/trino/values.yaml [hunk 1]
```diff
@@ -69,3 +69,5 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+podLabels: {}
```
Comment (hashhar) [inline charts/trino/values.yaml:73]: add a newline at end of file

**Comment by hashhar** [comment]
cc: @aalbu @nineinchnick

**Comment by cmanzur** [comment]
I already signed the license and sent it via email.

File: charts/trino/values.yaml [hunk 2]
```diff
@@ -38,7 +38,7 @@ additionalNodeProperties: {}
 
 additionalJVMConfig: {}
 
-additionalConfigProperties: {}
+additionalConfigProperties: []
```
Comment (thof) [inline charts/trino/values.yaml:41]: Why did you change the type for this field?

**Comment by mosabua** [comment]
Replaced by #75 ?

**Comment by mosabua** [comment]
This is now replaced by the merged PR #75
